src/connections/ResetPassword.ts

```diff
@@ -0,0 +1,61 @@
+import { createSelector } from "reselect";
+
+import {
+  requestPasswordResetFetchFailed,
+  requestPasswordResetIsFetching,
+  resetPasswordFetchFailed,
+  resetPasswordIsFetching
+} from "@/generated/v3/userService/userServicePredicates";
+import { ApiDataStore, PendingErrorState } from "@/store/apiSlice";
+import { Connection } from "@/types/connection";
+import { connectionHook } from "@/utils/connectionShortcuts";
+
+export const selectResetPassword = (store: ApiDataStore) => Object.values(store.logins)?.[0]?.attributes;
```

Comment:
As noted in the microservices PR, this resource should not be getting stored under a different resource type.